### ISSUE TITLE
About Your Experience

### ISSUE THREAD

**Issue by prp1277** [spences10/ama#3]
Hi Scott,

My name is Patrick Powell, I was checking the documentation for the VSCode VBA Extension when I stumbled across your blog and story. Although you have about 9 years of VBA experience on me, I think I am in a position similar to yours at the end of 2016 / beginning of 2017. 

I have a list of Excel and VBA resources that I use all the time when I run into problems, but getting used to the Github/developer environment has been an experience in and of itself. I wouldn't consider what I've done in my previous jobs _developing_ although I've created VBA modules in Excel and supplemented them with OneNote files. 

I guess my question is at what point did you consider yourself to be a developer as opposed to analyst?

**Comment by spences10** [comment]
Hi @prp1277,

Thanks for reading the blog I hope it was helpful for you.

**TL;DR:** depends on who I'm talking to, I considered myself a developer when at Barclays, even though I was an analyst developer.

From my early understanding of what a developer and an analyst developer were I have always considered myself an analyst developer.

This was my understanding:

* **Developer:** someone that worked from a spec and was confined to that one piece with no interaction with the business.

* **Analyst developer:** someone that sat with the business to understand what issue they were facing and came up with a solution for it, develoed it and implemented it.

I have swung between the two over the years and if people ask me what I do I say I'm a software developer or "I make Excel do stuff for people" or now "I make websites"
